app/controllers/api/v1/quotes_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Api::V1::QuotesController < ActionController::Base
+
+  def create
+    @quote = Quote.new quote_params
+    if @quote.save
+      render :show, status: :created, format: :json
+    else
+      render json: { message: @quote.errors }, status: 400
+    end
+  end
+
+  private
+
+  def quoted_person
+    return unless params[:quote][:quoted_person]
```

Comment:
this might read better as 
```ruby
return unless person = params[:quote][:quoted_person]
```
but not worth changing. Just makes it a bit more clear.